### PR TITLE
[MIG] sale_global_discount: Migration to 13.0

### PR DESCRIPTION
**Pull request by chienandalu**
Standard migration adapting tests. Fixes and forwards ports were needed. This PR depends on:

- [x] `base_global_discount`: OCA/server-backend#121
- [x] `account_global_discount` https://github.com/OCA/account-invoicing/pull/919

cc @Tecnativa TT26936

please review @pedrobaeza @carlosdauden

### CODE REVIEW

File: sale_global_discount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Global Discount",
+    "version": "13.0.1.0.0",
+    "category": "Sales Management",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "license": "AGPL-3",
+    "depends": ["account_global_discount"],
```
Comment (victoralmau) [inline sale_global_discount/__manifest__.py]:
```suggestion
    "depends": ["account_global_discount", "sale"],
```

Comment (chienandalu) [reply sale_global_discount/__manifest__.py]: sure, changed :+1:

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please fix pre-commit

**Review by manuelcalerosolis — APPROVED** [review]
Functional review!

**Review by dani-xtendoo — APPROVED** [review]
I have tested the module and the operation is correct

**Pull request merged**